lib/net/ssh/transport/session.rb

```diff
@@ -61,12 +61,12 @@ def initialize(host, options={})
       @bind_address = options[:bind_address] || nil
       @options = options
 
-      debug { "establishing connection to #{@host}:#{@port}" }
-
       @socket =
         if (factory = options[:proxy])
+          debug { "establishing connection to proxy" }
```

Comment:
> Nope, it doesn't make sense to place @host or @port because the proxy config ignores those settings entirely.

Does it ignores it? It does connect to there but through a proxy doesn't it?  What about something like:

```
establishing connection to #{@host}:#{@port} through proxy
```